### PR TITLE
Changed command to install python virtualenv

### LINKED ISSUE #132: Installing virtualenv through pip3 doesn't add command to $PATH
In `INSTALL.md` in section: `2. Python virtualenv`
Installing virtualenv through pip3 on ubuntu LTS causes 2 problems:

1. `virtualenv` doesn't get added to $PATH, so running:
```bash
$ virtualenv
zsh: command not found: virtualenv
```
2. If virtualenv is install through both `pip3` and `apt` (I installed it through `apt` later), the following error occurs:
```bash
$ virtualenv venv
AttributeError: module 'virtualenv.create.via_global_ref.builtin.cpython.mac_os' has no attribute 'CPython2macOsArmFramework'
```
---------------
#### So installing `virtualenv` through:
```bash
sudo apt install -y python3-virtualenv
```
is best practice **and not through pip3**

### PR DESCRIPTION
**Pull request by mina-andajos**
fixes #132

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -21,9 +21,10 @@ Please follow each part of this documentation in order to run your own instance
 2. Python virtualenv
 
     To install Python requirements in a virtual environment:
+
+    For Ubuntu, Debian derivatives:
```
Comment (tristanlatr) [inline INSTALL.md:25]:
```suggestion
    For Ubuntu, Debian derivatives (for macOS, you should be able to use `pip`):
```

**Comment by tristanlatr** [comment]
Looks good, thanks. Only a minor suggestion.

File: INSTALL.md (hunk 1 above)
Comment (mina-andajos) [reply INSTALL.md]: Ammended it 👍

**Comment by mina-andajos** [comment]
Changed it 👍.

File: INSTALL.md [hunk 2]
```diff
@@ -52,7 +53,7 @@ Please follow each part of this documentation in order to run your own instance
 
 ## .env settings
 
-Saskatoon uses `python-dotenv` to manage local environment settings. Copy the [saskatoon/env.template](saskatoon/env.template) file into `saskatoon/.env` and adapt it to your needs. 
+Saskatoon uses `python-dotenv` to manage local environment settings. Copy the [saskatoon/env.template](saskatoon/env.template) file into `saskatoon/.env` and adapt it to your needs.
```
Comment (2ynn) [inline INSTALL.md:56]: @mina-andajos can we change this line for `django-dotenv` as discussed in #137 ?

Also would it be okay to rebase these changes onto develop?

Comment (mina-andajos) [reply INSTALL.md:56]: will do 👍

Comment (mina-andajos) [reply INSTALL.md]: changed it and rebased 👍

**Comment by tristanlatr** [comment]
Thanks @mina-andajos,

This PR looks perfect, sorry about the duplicate work, I had to edit the setup.py locally unless it would not run. 

I think we can merge this PR.

**Review by 2ynn — APPROVED** [review]

**Pull request merged**